eclipse-scout-core/src/tabbox/SimpleTab.less

```diff
@@ -84,8 +84,8 @@
           line-height: @font-size-small;
```

Comment:
line-height and min-width leads to misalignment in the scout theme. Maybe use the same style as for the chat menu?
Also, I think display: inline-block, margin-top and margin-bottom have no effect and can be removed, or am I wrong?